compiler/test/compilable/user_defined_attributes.d

```diff
@@ -0,0 +1,18 @@
+
+enum Test;
+
+@true @null @byte int x;
+@(int) int y;
+@"test" @`test2` void f();
```

Comment:
Need tests for @1, @'a'

Reply:
I.e. anything accepted as a `TemplateArgument`

Reply:
Done